network-api/networkapi/templates/fragments/donate_banner.html

```diff
@@ -13,7 +13,7 @@
 
                 {% block left_banner_column %}
                     <div class="tw-w-6/12 tw-m-auto large:tw-order-2">
-                        <img src="{{ background_image.url }}" alt="Banner Image" width="auto" height="auto">
```

Comment:
`auto` is not a valid value. 

The static `alt` does not add any descriptive information to screen reader users.